modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-serial-console_{context}"]
```

Comment:
not sure if we can have `{}` in an ID like that :thinking:

Reply:
You mean the `:boot:` attribute? I've done it before and it doesn't seem to cause any issues 😬